x-pack/libbeat/docs/aws-credentials-config.asciidoc

```diff
@@ -85,12 +85,12 @@ aws> sts get-session-token --serial-number arn:aws:iam::1234:mfa/your-email@exam
 ----
 
 Because temporary security credentials are short term, after they expire, the user needs to generate new ones and modify
-the aws.yml config file with the new credentials. Unless https://www.elastic.co/guide/en/beats/metricbeat/current/_live_reloading.html[live reloading]
-feature is enabled for Metricbeat, the user needs to manually restart Metricbeat after updating the config file in order
-to continue collecting Cloudwatch metrics. This will cause data loss if the config file is not updated with new
-credentials before the old ones expire. For Metricbeat, we recommend users to use access keys in config file to enable
+the aws.yml config file with the new credentials. Unless https://www.elastic.co/guide/en/beats/{beatname_uc}/current/_live_reloading.html[live reloading]
```

Comment:
I would like to get @bmorelli25 thoughts on this too. I think for this URL to work for all the Beats docs, you need to add the `:beatname_url: {beats-ref-root}/{beatname_lc}/{branch}` attribute to the top of the file and then update the URL to `{beatname_url}/_live_reloading.html[live reloading]`.

Reply:
Yup, that's exactly correct. @mjmbischoff, and example can be seen here https://github.com/elastic/beats/blame/aa21b46788af5ef93cd73cea32c990fa7bdb19f6/libbeat/docs/tab-widgets/set-connection.asciidoc#L3. Usage is on L17.

Reply:
Updated in 4f3c178.